src/StartProject.Asp/StartProject.Asp.csproj

```diff
@@ -6,7 +6,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-      <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="5.0.13" />
+      <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="6.0.0" />
```

Comment:
should we add this also to the standard targets file, since it also depends on the net version?

Reply:
Yes we could do this, howevr its only purpose is to simplify development with razor pages, there is no actual reason to have it in a real application. I wanted to remove it first but decided otherwise for some reason 🤷‍♂️